app/scripts/dimApp.i18n.js

```diff
@@ -500,35 +500,171 @@
+            ShowAllConfigs: "Afficher toutes les configurations",
+            ShowConfigs: "Afficher les configurations",
+            HideAllConfigs: "Cacher toutes les configurations",
+            HideConfigs: "Cacher les configurations",
+            Loading: "Chargement des meilleurs sets",
+            Vendor: "Inclure articles",
+            Exclude: "Articles exclus",
+            ExcludeHelp: "Shift + cliquez un article (ou glissez-déposez dans cet espace) pour construire des sets sans équipement spécifique.",
+            LockPerk: "Verrouiller amélioration",
+            Missing1: 'Pièces légendaires ou éxotiques manquantes pour construire un set complet!',
+            Missing2: 'Pièces légendaires ou éxotiques manquantes pour construire un set complet!'
+          },
           Loadouts: {
+            Loadouts: "Loadouts",
+            Before: 'Avant \'{{name}}\'',
             Create: "Créer Loadout",
             FromEquipped: "Equipé",
             Edit: "Modifier Loadout",
             Delete: "Effacer Loadout",
+            ConfirmDelete: "Êtes-vous sûr de vouloir supprimer '{{name}}'?",
             ApplySearch: "Articles = \"{{query}}\"",
             MaximizeLight: "Maximiser la Lumière",
             ItemLeveling: "Evolution d'article",
             GatherEngrams: "Rassembler les Engrammes",
             GatherEngramsExceptExotics: "Exotiques",
             RestoreAllItems: "Tous les Articles",
-            Loadouts: "Loadouts",
+            LoadoutName: "Nom du loadout...",
+            Save: "Sauvegarder",
+            SaveAsNew: "Sauvegarder comme nouveau",
+            Cancel: "Annuler",
+            ItemsWithIcon: "Les articles avec cet icone seront équipés.",
+            ClickToEquip: "Cliquez sur un bouton d'article pour l'équiper.",
             AppliedAuto: "Constructeur de Loadout automatique",
             Applied: "Votre loadout de {{amount}} articles a été transféré à votre {{store}}",
             AppliedError: "Aucun des articles de votre loadout n'a pu être transféré.",
             AppliedWarn: "Votre loadout a été partiellement transféré, mais {{failed}} sur {{total}} articles ont échoué." },
+          Manifest: {
+            Build: "Base de données d'info de Destiny en cours de construction",
+            Download: "Téléchargement en cours des derniers infos Destiny de Bungie",
+            Error: "Erreur de chargement des infos Destiny:\n{{error}}\nRedémarrez pour rééssayer.",
+            Outdated: "Infos Destiny expirées",
+            OutdatedExplanation: "Bungie a mis à jour les base de données de Destiny. Redémarrez DIM pour charger les nouvelles informations. Notez que certaines fonctionalités de DIM puissent ne pas fonctionner pendant quelques heures après que Bungie mette à jour Destiny, le temps que les nouvelles données se propagent dans leur système.",
+            BungieDown: "Bungie.net rencontre des problèmes.",
+            Load: "Chargement des données sauvegardées de Destiny",
```

Comment:
Can you please add LoadCharInv: "Loading Destiny characters and inventory", here with the correct translation, do not add it to the english definitions though.